client/src/components/CoverPhoto.tsx

```diff
@@ -40,7 +42,12 @@ const CoverPhoto = ({
       style={{ '--cover-photo--size': size } as StyleProps}
     >
       {image ? (
-        <LazyImage className="aspect-square object-cover" src={image.url} />
+        <LazyImage
+          className={cx('aspect-square object-cover', {
+            'opacity-100': !ease,
```

Comment:
I wonder if we should go a level deeper and add an `animate` or `animateIn` prop on the `LazyImage` component (assuming this is used to prevent that fade-in effect). Thoughts?

Reply:
Makes sense, also updated in https://github.com/apollographql/spotify-showcase/pull/87/commits/7b70a9dbda1107d0b88581a2203bdd8e993ba0c4